### PR TITLE
Docs/windows installer

### PR DESCRIPTION
**Pull request by russcam**
Updated documentation for the Windows Installer 6.0 release:

- new images
- additional information relating to the new X-Pack screen and license options

@Mpdreamz would you mind specifically double-checking the instructions with regards to `INSTALLDIR` from the command line?

### CODE REVIEW

File: docs/reference/setup/install/windows.asciidoc [hunk 1]
```diff
@@ -369,11 +435,16 @@ The `.msi` can also upgrade Elasticsearch using the command line.
...
+used at first install time; the Windows Installer does not remember these properties. 
 
 For example, if you originally installed with the command line options `PLUGINS="x-pack"` and
 `LOCKMEMORY="true"`, then you must pass these same values when performing an
 upgrade from the command line.
+
+The **exception** to this is `INSTALLDIR` (if originally specified), which must be a different directory to the 
+current installation. 
+If setting `INSTALLDIR`, we recommend that the final directory in the path be the version of Elasticsearch e.g.
+++C:\Program Files\Elastic\Elasticsearch\{version}++
```
Comment (Mpdreamz) [inline docs/reference/setup/install/windows.asciidoc]: Not just recommended but currently we have to mandate the installer ends in the version when running the silent install otherwiae the install will fail. THE UI can cope with this though.

**Review by Mpdreamz — CHANGES_REQUESTED** [review]
Awesome! Left a small comment around INSTALLDIR

File: docs/reference/setup/install/windows.asciidoc (hunk 1 above)
Comment (russcam) [reply docs/reference/setup/install/windows.asciidoc]: I think the installer should only mandate that the directory for the upgrade version is different to the existing version, and not a child directory of the existing version.

**Comment by russcam** [comment]
I updated the description for `INSTALLDIR` when upgrading. Also highlighted that the `INSTALLDIR` path final directory must be the version number.

**Review by Mpdreamz — APPROVED** [review]

**Pull request merged**